src/commands/synthetics/README.md

```diff
@@ -178,8 +180,14 @@ All options under the `config` key allow overriding the configuration of the tes
   - `count`: (integer) number of attempts to perform in case of test failure.
   - `interval`: (integer) interval between the attempts (in milliseconds).
 - `startUrl`: (string) new start URL to provide to the test.
+- `startUrlSubstitutionRegex`: (string) regex to modify the starting URL of the test (browser and HTTP tests only), whether it was given by the original test or by the configuration override `startUrl`. If the URL contains variables, this regex will be applied after the interpolation of the variables. The format is `s/your_regex/your_substitution/modifiers`, for instance `s/(https://www.)(.*)/$1extra-$2/` to transform `https://www.example.com` into `https://www.extra-example.com`.
```

Comment:
We can add a quick mention that it should follow JS regex syntax, I think there are some differences between regex language flavors. Maybe we should also move the mention of variables below in the deprecated part.

Reply:
I'll clarify the flavor indeed but the field override `variables` itself is not deprecated, it's the "URL variables" that you could use in your `startingUrl`.